fitbenchmarking/fitbenchmarking
# Add documentation for minimizer tags

#### Description of Work

Fixes #828 


#### Testing Instructions

1. Proof read docs, make sure they are clear and make sense
2.
3.

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
